libs/application/templates/secondary-school/src/fields/overview/CustodianOverview.tsx

```
@@ -57,17 +66,20 @@ export const CustodianOverview: FC<FieldBaseProps> = ({
           ? overview.custodian.subtitle
           : overview.otherContact.subtitle,
       )}
-      isEditable={application.state === States.DRAFT}
+      isEditable={isEditable}
     >
       <Box>
-        {!!custodiansExternalData.length && (
+        {showCustodians && (
           <GridRow>
             {custodiansExternalData.map((custodian, index) => (
               <GridColumn span="1/2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)